include/z64message.h

```diff
@@ -83,67 +83,67 @@ typedef enum {
+    /* 0x02 */ MASK_REACTION_SET_CARPENTER_2,
+    /* 0x03 */ MASK_REACTION_SET_CARPENTER_3,
+    /* 0x04 */ MASK_REACTION_SET_CARPENTER_4,
+    /* 0x05 */ MASK_REACTION_SET_HYRULIAN_GUARD,
+    /* 0x06 */ MASK_REACTION_SET_HEISHI4_1,
+    /* 0x07 */ MASK_REACTION_SET_HEISHI4_2,
+    /* 0x08 */ MASK_REACTION_SET_CUCCO_LADY,
+    /* 0x09 */ MASK_REACTION_SET_CARPENTERS_SON,
+    /* 0x0A */ MASK_REACTION_SET_KAKARIKO_ROOF_MAN,
+    /* 0x0B */ MASK_REACTION_SET_WINDMILL_MAN,
+    /* 0x0C */ MASK_REACTION_SET_12, // Unused
+    /* 0x0D */ MASK_REACTION_SET_CURSED_SKULLTULA_MAN,
+    /* 0x0E */ MASK_REACTION_SET_DAMPE,
+    /* 0x0F */ MASK_REACTION_SET_GRAVEYARD_KID,
+    /* 0x10 */ MASK_REACTION_SET_SARIA,
+    /* 0x11 */ MASK_REACTION_SET_MIDO,
+    /* 0x12 */ MASK_REACTION_SET_FADO,
+    /* 0x13 */ MASK_REACTION_SET_KOKIRI_1,
+    /* 0x14 */ MASK_REACTION_SET_KOKIRI_2,
+    /* 0x15 */ MASK_REACTION_SET_SKULL_KID,
+    /* 0x16 */ MASK_REACTION_SET_ZELDA,
+    /* 0x17 */ MASK_REACTION_SET_MALON,
+    /* 0x18 */ MASK_REACTION_SET_TALON,
+    /* 0x19 */ MASK_REACTION_SET_INGO,
+    /* 0x1A */ MASK_REACTION_SET_LAKESIDE_PROFESSOR,
+    /* 0x1B */ MASK_REACTION_SET_MAGIC_BEAN_SALESMAN,
+    /* 0x1C */ MASK_REACTION_SET_RUNNING_MAN,
+    /* 0x1D */ MASK_REACTION_SET_ZORA,
+    /* 0x1E */ MASK_REACTION_SET_KING_ZORA,
+    /* 0x1F */ MASK_REACTION_SET_RUTO,
+    /* 0x20 */ MASK_REACTION_SET_GORON,
+    /* 0x21 */ MASK_REACTION_SET_DARUNIA,
+    /* 0x22 */ MASK_REACTION_SET_GERUDO_WHITE,
+    /* 0x23 */ MASK_REACTION_SET_NABOORU,
+    /* 0x24 */ MASK_REACTION_SET_DANCING_COUPLE,
+    /* 0x25 */ MASK_REACTION_SET_37, // ENHY_TYPE_AOB
+    /* 0x26 */ MASK_REACTION_SET_38, // ENHY_TYPE_COB
+    /* 0x27 */ MASK_REACTION_SET_39, // ENHY_TYPE_AHG_2
+    /* 0x28 */ MASK_REACTION_SET_40, // ENHY_TYPE_BOJ_3
+    /* 0x29 */ MASK_REACTION_SET_41, // ENHY_TYPE_AHG_4
+    /* 0x2A */ MASK_REACTION_SET_42, // ENHY_TYPE_BOJ_5
+    /* 0x2B */ MASK_REACTION_SET_43, // ENHY_TYPE_BBA
+    /* 0x2C */ MASK_REACTION_SET_44, // ENHY_TYPE_BJI_7
+    /* 0x2D */ MASK_REACTION_SET_45, // ENHY_TYPE_CNE_8
+    /* 0x2E */ MASK_REACTION_SET_46, // ENHY_TYPE_BOJ_9
+    /* 0x2F */ MASK_REACTION_SET_47, // ENHY_TYPE_BOJ_10
+    /* 0x30 */ MASK_REACTION_SET_48, // ENHY_TYPE_CNE_11
+    /* 0x31 */ MASK_REACTION_SET_49, // ENHY_TYPE_BOJ_12
+    /* 0x32 */ MASK_REACTION_SET_50, // ENHY_TYPE_AHG_13
+    /* 0x33 */ MASK_REACTION_SET_51, // ENHY_TYPE_BOJ_14
+    /* 0x34 */ MASK_REACTION_SET_52, // ENHY_TYPE_BJI_15
+    /* 0x35 */ MASK_REACTION_SET_53, // ENHY_TYPE_BOJ_16
+    /* 0x36 */ MASK_REACTION_SET_54, // ENHY_TYPE_AHG_17
+    /* 0x37 */ MASK_REACTION_SET_55, // ENHY_TYPE_BOB_18
+    /* 0x38 */ MASK_REACTION_SET_56, // ENHY_TYPE_BJI_19
+    /* 0x39 */ MASK_REACTION_SET_57, // ENHY_TYPE_AHG_20
+    /* 0x3A */ MASK_REACTION_SET_HAGGLING_TOWNSPEOPLE_1,
+    /* 0x3B */ MASK_REACTION_SET_HAGGLING_TOWNSPEOPLE_2,
+    /* 0x3C */ MASK_REACTION_SET_MAX
 } FaceReactionSet;
```

Comment:
enum name, and the one at the top
btw oot still has a bunch of anonymous enums, so its not very common to list the name at the top. But I think, like MM, it should be changed eventually. So I wont request that it be removed. Just something to keep in mind.

Reply:
sorry, i commented on an earlier commit and it was fixed later. My comment about anonymous enums still applies, but no action is needed

Reply:
Cool, I wasn't sure if OoT wanted to shift to that eventually but good to know